t/ts/admin_api.ts

```diff
@@ -14,22 +14,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import axios, { type Method, type RawAxiosRequestHeaders } from 'axios';
+import axios, {
+  type AxiosRequestConfig,
+  type Method,
+  type RawAxiosRequestHeaders,
+} from 'axios';
 
 export const request = async (
-  uri: string,
+  url: string,
   method: Method = 'GET',
   body?: object,
   headers?: RawAxiosRequestHeaders,
+  config?: AxiosRequestConfig,
 ) => {
   return axios.request({
     method,
     // TODO: use 9180 for admin api
-    url: `http://127.0.0.1:1984/${uri}`,
+    baseURL: 'http://127.0.0.1:1984',
+    url,
```

Comment:
i can understand this change, but where is it used? 🤔

Reply:
https://github.com/apache/apisix/pull/12224/files#diff-3701f25e61c6b3422208bbab8c63eee7f5e5315970672229ce0e5c5965f8a280R20